templates/api/clients/ruby/Gemfile.lock.tpl

```diff
@@ -1,27 +1,29 @@
 {{- $_ := stencil.ApplyTemplate "skipGrpcClient" "ruby" -}}
+{{ file.Static }}
 PATH
   remote: .
   specs:
-    {{ .Config.Name }}_client (1.63.0)
-      grpc (~> 1.38)
+    {{ .Config.Name }}_client (0.0.0)
+      grpc (~> 1.59)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    google-protobuf (3.19.1)
-    googleapis-common-protos-types (1.3.0)
-      google-protobuf (~> 3.14)
-    grpc (1.42.0)
+    google-protobuf (3.25.0)
+    google-protobuf (3.25.0-arm64-darwin)
+    googleapis-common-protos-types (1.10.0)
       google-protobuf (~> 3.18)
+    grpc (1.59.2)
+      google-protobuf (~> 3.24)
       googleapis-common-protos-types (~> 1.0)
-    rake (13.0.6)
-
+    rake (13.1.0)
 PLATFORMS
   -darwin-20
+  arm64-darwin-22
 
 DEPENDENCIES
   {{ .Config.Name }}_client!
   rake
 
 BUNDLED WITH
-   2.2.29
+   2.3.26
```

Comment:
thought: I wonder if it would be worth putting the bundler version in the manifest.

Reply:
I honestly don't know, but given we're unmanaging it, I would think once is okay?

Reply:
Oh right, ignore this then